openquakeplatform/static/geonode/css/oqplatform_leaflet.css

```diff
@@ -1,5 +1,10 @@
+#wrap,
+#wrap .container {
+    width: 100%;
+}
 #oq-page-middle {
-    background: #555555;
+    background: #fff;
```

Comment:
Could we continue to use 6 chars syntax for `#rrggbb` colors ?

Reply:
Fixed here: https://github.com/gem/oq-platform2/pull/16/commits/63fbc4a24a06952253b7cad96c4898adb28970c4

Reply:
Fixed here: https://github.com/gem/oq-platform2/pull/16/commits/a5bb6bdd11c8f111a993446fc3b83f86ee5fe9b4